amis/packer_alinux.json

```diff
@@ -173,7 +173,8 @@
     {
       "type" : "shell",
       "inline" : [
-        "curl --retry 3 -L https://www.chef.io/chef/install.sh | sudo bash -s -- -v {{user `chef_version`}}"
+        "_region={{user `region`}}",
```

Comment:
two things:
* quote assignment, e.g. `"_region=\"{{user `region`}}\"",`
* I would use `region` instead of `_region`, to be consistent with other places in this same file where `region` is defined as well